mm2src/lp_ordermatch.rs

```diff
@@ -2956,16 +2973,17 @@ async fn check_balance_for_maker_orders(ctx: MmArc, ordermatch_ctx: &OrdermatchC
 async fn handle_timed_out_maker_matches(ctx: MmArc, ordermatch_ctx: &OrdermatchContext) {
     let now = now_ms();
     let storage = MyOrdersStorage::new(ctx.clone());
-    let mut my_maker_orders = ordermatch_ctx.my_maker_orders.lock().await;
+    let mut my_maker_orders = ordermatch_ctx.my_maker_orders.lock().clone();
 
     for (_, order) in my_maker_orders.iter_mut() {
```

Comment:
As I can see, `iter_mut` might be not required here anymore. Could you check it, please?